spanner/api/Spanner.Samples/InsertSampleDataAsync.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_data]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class InsertSampleDataAsyncSample
+{
+    public async Task InsertSampleDataAsync(string projectId, string instanceId, string databaseId)
+    {
+        const int firstSingerId = 1;
+        const int secondSingerId = 2;
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+        List<Singer> singers = new List<Singer>
```

Comment:
Why are we defining the data here? If we already have the Singer, Album classes, couldn't we just take the list of singers and albums as parameters? And hopefully we can reduce this sample to jus inserting in one table.

Reply:
This one seems to be one of the ones used in the Getting Started guides, so if that's the case, it's fine to leave as is.